src/dragonfly/dragonfly/pump.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+import sys
+import argparse
+import time
+import rclpy
+from rclpy.node import Node
+from dragonfly_messages.srv import Pump
+import rx
+import rx.operators as ops
+from rx.scheduler import NewThreadScheduler
+
+
+class BagInflateService(Node):
+    def __init__(self, arg_id, sim=False):
+        super().__init__('bag_inflate_service')
+        self.id = arg_id
+        self.sim = sim
+        if not self.sim:
```

Comment:
Sweet!  add this to the docker startup? https://github.com/BCLab-UNM/dragonfly-controller/blob/master/docker/entrypoint.py#L10-L12

Reply:
Oh, and I tried this out on the dragonfly, add a default to the sim parameter or add it to the various scripts.